### PR TITLE
Update task_set.json

### PR DESCRIPTION
**Pull request by gchhablani**
This PR adds new tasks. 

Additionally, I want to add task descriptions for the options too. I'm not sure how would that be possible right now, but if it is needed, I'd love to look into it.

Thanks,
Gunjan

### CODE REVIEW

File: task_set.json [hunk 1]
```diff
@@ -76,11 +144,5 @@
             "sentiment-scoring",
             "other"
         ]
-    },
-    "other": {
```
Comment (theo-m) [inline task_set.json:80]: I wouldn't remove "other" just yet, it's hard to guarantee we currently have exhausted all the possible tasks :)

Comment (gchhablani) [reply task_set.json:80]: "other"  is in between. This is a sorted JSON.  :P

File: task_set.json [hunk 2]
```diff
@@ -1,71 +1,139 @@
...
+    },
+    "image-classification": {
+        "description": "image classification tasks",
+        "options": [
+            "multi-class-classification",
+            "multi-label-classification",
+            "other"
+        ]
+    },
+    "other": {
```
Comment (gchhablani) [inline task_set.json:42]: @theo-m

**Review by theo-m — COMMENTED** [review]
Looks good! If you feel like adding descriptions it's very welcome, it wouldn't require all that much to welcome them in the app, but it's a lot of work to do. 
If you want to start piece by piece and split up work it's fine for me, e.g. the following schema:
```
options: (id: string, description: string | null)[]
```
and we iteratively fill the description until none is null anymore.

**Comment by gchhablani** [comment]
HI @theo-m 
I'll start adding the descriptions one by one :)
Should I do it in the same PR?

**Comment by theo-m** [comment]
You can work the file on this repo, but I've actually reverted the dependencies, see https://github.com/huggingface/datasets/pull/2107, so that the PR should now be in the `datasets` repo ;)

**Comment by lewtun** [comment]
@lhoestq can we close this since the tasks are now stored in `datasets` ([link](https://github.com/huggingface/datasets/blob/master/src/datasets/utils/resources/tasks.json))?

**Comment by lhoestq** [comment]
Yes indeed !

All those new tasks could be added directly in [tasks.json](https://github.com/huggingface/datasets/blob/master/src/datasets/utils/resources/tasks.json) in `datasets` though :)

**Pull request closed without merging**